Update docs for MC-1225 (#142)
hazelcast/management-center-docs: docs/modules/deploy-manage/pages/historical-metrics.adoc

`Data point memory compression ratio` shows how many times less disk space is used than in-memory storage.

== Disabling Metrics Persistence
If you don't want metrics to be persisted on disk, set the xref:system-properties.adoc#hazelcast-mc-metrics-persistence-enabled[`hazelcast.mc.metrics.persistence.enabled`] property to `false`, which stores metrics only in memory.

> TIP: Disabling persistence allows Management Center to run on an operating system that doesn't support RocksDB. See the xref:../../getting-started/pages/overview.adoc#os-requirements[Requirements section] for details.

Reviewer: ```suggestion
TIP: Disabling persistence allows Management Center to run on an operating system that doesn't support RocksDB. See the xref:getting-started:overview.adoc#os-requirements[Requirements section] for details.
```

- Decrease the amount of metrics that the members send. See the xref:{page-latest-supported-hazelcast}@hazelcast:maintain-cluster:monitoring.adoc[Metrics section] of the Hazelcast documentation for details about configuring metrics collection.
- <<changing-the-metrics-time-to-live, Decrease the TTL of persisted metrics>>.

== Disabling Metrics Persistence

> Disabling Metrics Persistence allows MC to run on OS that aren't supported by RocksDB. Metrics are stored only in memory.

Reviewer: ```suggestion
If you don't want metrics to be persisted on disk, set the xref:system-properties.adoc#hazelcast-mc-metrics-persistence-enabled[`hazelcast.mc.metrics.persistence.enabled`] property to `false`, which stores metrics only in memory.

TIP: Disabling persistence allows Management Center to run on an operating system that doesn't support RocksDB.
```

Can we move this above the 'Troubleshooting' section?

> To disable metrics persistence:

Reviewer: ```suggestion
```

> Set the xref:system-properties.adoc#hazelcast-mc-metrics-persistence-enabled[`hazelcast.mc.metrics.persistence.enabled`] property to `false`.

Reviewer: ```suggestion
```

- Decrease the amount of metrics that the members send. See the xref:{page-latest-supported-hazelcast}@hazelcast:maintain-cluster:monitoring.adoc[Metrics section] of the Hazelcast documentation for details about configuring metrics collection.
- <<changing-the-metrics-time-to-live, Decrease the TTL of persisted metrics>>.

== Disabling Metrics Persistence

> Set the xref:system-properties.adoc#hazelcast-mc-metrics-persistence-enabled[`hazelcast.mc.metrics.persistence.enabled`] property to `false`.

Reviewer 2: please add more details like: 
- it allows MC to run on OS that aren't supported by RocksDB;
- metrics are stored only in memory with this property;
